yarn.lock

```diff
@@ -4737,6 +4737,13 @@ bytes@3.1.2:
   resolved "https://registry.yarnpkg.com/bytes/-/bytes-3.1.2.tgz#8b0beeb98605adf1b128fa4386403c009e0221a5"
   integrity sha512-/Nf7TyzTx6S3yRJObOAV7956r8cr2+Oj8AC5dt8wSP3BQAoeX58NoHyCU8P8zGkNXStjTSi6fzO6F0pBdcYbEg==
 
+c-kzg@^1.0.9:
```

Comment:
is it intended that `c-kzg` is added in yarn.lock? I noticed it was showing up as a diff wenn running `yarn install` locally, this is fixed but this change seems unrelated to the PR

Reply:
yeah, its unrelated to this PR